.github/workflows/package-validation.yml

```
@@ -266,7 +266,19 @@ jobs:
         run: npm run build
 
       - name: Build broker binary
-        run: cargo build --release --bin agent-relay-broker
+        # Pin the broker's reported version to the same release line as
+        # `agent-relay` / `@agent-relay/sdk`. `option_env!` in
+        # crates/broker/src/util/version.rs consumes this at compile time
+        # and falls back to `CARGO_PKG_VERSION` when unset.
+        env:
+          AGENT_RELAY_VERSION: ${{ needs.build.outputs.new_version }}
```

actionlint (1.7.12)

[error] property "build" is not defined in object type {changes: {outputs: {web_only: string}; result: string}}

(expression)